pkg/p2p/p2p.go

```diff
@@ -27,6 +26,11 @@ type Stream interface {
 	io.Closer
 }
 
+// PeerSuggester suggests a peer to retrieve a chunk from
+type PeerSuggester interface {
+	SuggestPeer(addr []byte) (peerAddr string, err error)
```

Comment:
I think that it is better to deal with swarm.Address type for addresses. Something like:

```
SuggestPeer(chunk swarm.Address) (peer swarm.Address, err error)
```